config.env-template

```diff
@@ -3,31 +3,41 @@ PGHOST=
 PGUSER=
 PGPASSWORD=
 PGPASS=
+PGPORT=
+POSTGRES_PORT_5432_TCP_PORT=
```

Comment:
Appears to be unused?

Reply:
Used within the mdillon:postgis container used in the default development version ([see](https://hub.docker.com/r/mdillon/postgis/) )